.github/labeler.yml

```diff
@@ -5,3 +5,6 @@
 
 i18n:
 - any: ['src/pages/**/*.md', '!src/pages/en/**/*']
+
+hacktoberfest-accepted:
```

Comment:
how does this work? does it append PRs merged to the end of the file, or does it auto-label PRs to the file as `hacktoberfest-accepted`?

Reply:
This auto-labels PRs that change that file so we don’t forget. We could also do this manually obviously but thought this might be safest (as long as it doesn’t prompt spam?)